webui/react/src/services/stream/wire.ts

```diff
@@ -0,0 +1,100 @@
+// Code generated by stream-gen. DO NOT EDIT.
```

Comment:
remember to mark this as ignored for eslint and prettier, and mark it as generated in `.gitattributes`

Reply:
Good catch, thanks!